app/components/content/talk_to_us_component.html.erb

```diff
@@ -1,21 +1,29 @@
 <div class="talk-to-us-content">
```

Comment:
@gazcarr @philspray Did we want to remove the Talk to us component from this landing page, to put more focus on the mailing list CTAs?

Reply:
Yes I think that would make the page more focused.

Reply:
Agreed

Reply:
@ethax-ross Is this something you need to do (is it possible on this page?)?